src/App.js

```diff
@@ -29,6 +31,9 @@ import { SIGN_IN, EDIT_PROFILE, CONFIRM_TRANSACTION } from './modals'
 
 import './App.css'
 
+// Authereum integration
+injectWeb3('rinkeby')
```

Comment:
Can this be injected only at App.js? I think we need to inject somewhere we can pass the network name (eg: https://github.com/wearekickback/app/blob/dev/src/api/web3.js#L112)